aeron-driver/src/main/java/io/aeron/driver/IpcPublication.java

```diff
@@ -119,7 +130,11 @@ enum State
         this.unblockTimeoutNs = ctx.publicationUnblockTimeoutNs();
         this.untetheredWindowLimitTimeoutNs = params.untetheredWindowLimitTimeoutNs;
         this.untetheredRestingTimeoutNs = params.untetheredRestingTimeoutNs;
-        this.unblockedPublications = ctx.systemCounters().get(UNBLOCKED_PUBLICATIONS);
+        this.imageLivenessTimeoutNs = ctx.imageLivenessTimeoutNs();
+        final SystemCounters systemCounters = ctx.systemCounters();
+        this.unblockedPublications = systemCounters.get(UNBLOCKED_PUBLICATIONS);
+        this.errorMessagesReceived = systemCounters.get(ERROR_FRAMES_RECEIVED);
```

Comment:
I think we should have a separate counter for IPC rejections.  There are no actual error frames involved here, but we would still like some observability.